content/en/tracing/advanced/adding_metadata_to_spans.md

```diff
@@ -16,26 +16,18 @@ further_reading:
   text: "Explore your services, resources, and traces"
 ---
 
-Adding metadata in the form of key-value pairs to a span allows to correlate traces with other Datadog products to provide more details about specific spans. [Read more about tagging][1]
+Adding metadata in the form of key-value pairs to a span allows to correlate traces with other Datadog products to provide more details about specific spans. Metadata can be either added [to a single span](#adding-metadata-to-a-span) or [globally to all spans](#adding-metadata-globally-to-all-spans)
 
-Metadata can be:
-
-- [Adding tags to a span](#adding-tags-to-a-span)
-- [Adding tags globally to all spans](#adding-tags-globally-to-all-spans)
-- [Further Reading](#further-reading)
-
-## Adding tags to a span
+## Adding Metadata to a span
 
 {{< tabs >}}
 {{% tab "Java" %}}
 
-Tags are key-value pairs attached to spans. All tags share a single namespace.
+The Datadog UI uses specific metadata to set UI properties, such as an application's service name. A full list of these metadata can be found in the [Datadog][1] and [OpenTracing][2] APIs.
 
-The Datadog UI uses specific tags to set UI properties, such as an application's service name. A full list of these tags can be found in the [Datadog][1] and [OpenTracing][2] APIs.
+**Note**: Custom metadata are set using the OpenTracing API.
 
-Custom tags are set using the OpenTracing API.
-
-Custom tags may be set for auto-instrumentation by grabbing the active span out of the global tracer.
+Custom metadata may be set for auto-instrumentation by grabbing the active span out of the global tracer.
```

Comment:
Custom metadata may be set for auto-instrumentation by grabbing the active span out of the global tracer and setting a tag.